### ISSUE TITLE
How to add external urls?

### ISSUE THREAD

**Issue by ingeniapp** [amitdar/FullPageDashboard#3]
Hi,

I'm trying add a tab with a external url (not localhost) and I always get a blank tab.
Can I add a http://domain.com tab into Dashboard?

Thanks

**Comment by guysoft** [comment]
Have you tried using FullPageDasbhoard? it comes as the default option.

**Comment by guysoft** [comment]
You are supposed to get an message box to add the url. What do you get when you press the plus button inside the browser, not using a new Chromium tab?

**Comment by Brutalspark** [comment]
@guysoft I believe I have the same issue as @ingeniapp we get the message box to add the url. When we do so it accepts and creates a "tab" but when I select the tab it is simply a white page, no data displayed. This WILL display SOME websites, but not all.

**Comment by guysoft** [comment]
1. There is a known issue with https websites? Is that is?
2. I need specific examples.

**Comment by MichaelNewham** [comment]
@guysoft Yes, most 'https' websites that I've tried come up as blank tabs. Sites such as -

https://www.nytimes.com
https://www.irishtimes.com

But then this one works for some reason - 
https://darksky.net/forecast/53.0966,-7.9109/us12/en
____
**Can you point me in the direction of where I could change the default behaviour within the code to allow https sites appear?**

**Comment by guysoft** [comment]
@MichaelNewham Yes,. FullPageOS ships with FullPageDashboard, although its unmaintained, just no one offered an alternative.

The reason some pages work and some don,t is because there is some testing on the website side. I have no idea what.

**Comment by Berend020** [comment]
I confirm this problem. 
FullPageOS works great but this issue is bad experience for users.

**Comment by stappersg** [comment]
> this is bad experience for users

Baggers are not choosers.

**Comment by guysoft** [comment]
Indeed, this project is up for adoption at this point.

**Comment by fivestones** [comment]
> Baggers are not choosers.

True! They can't choose what to put in the bags. They just put all your things in and smile.
